docs-v2/pages/databases.mdx

```diff
@@ -0,0 +1,32 @@
+# Working with Databases
+Connecting to databases is a fundamental aspect of creating powerful workflows for your applications. Whether you're storing application data, querying user information, or analyzing event logs, most workflows require querying data at some step.
+
+Every Pipedream workflow is deployed to its own virtual machine in AWS. This means your workflow runs in its own execution environment with dedicated RAM and disk space, isolated from other users' workflows.
+
+However, outbound traffic from your workflows uses the same network as other AWS services in the us-east-1 region. **This means your workflows share AWS's `us-east-1` network, sending requests from standard AWS IP ranges.**
+
+## How to Connect to Restricted Databases
+When you need to connect to a database that restricts access to a specific set of IP addresses, you have a couple options:
```

Comment:
Consider adding an intro paragraph about database security for the unaware, then go into the specific options provided by Pipedream.

```suggestion
It's best practice to restrict which IP addresses can access your databases. This helps prevent unauthorized access by reducing the possible IP addresses that can make a connection. 

Typically this is enforced by a firewall, or a configuration within the database to only accept connections from trusted IP addresses.

Pipedream offers two options to facilitate secure connections:
```